hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -104,8 +116,12 @@
     MESSAGES_LOADING_WAIT_FOR_TOPICS_CACHE("frontend.messages.loading.wait.for.topics.cache", 10),
     MESSAGES_LOADING_WAIT_FOR_BROKER_TOPIC_INFO("frontend.messages.loading.wait.for.broker.topic.info", 5),
 
-    CONSUMER_COMMIT_OFFSET_PERIOD("consumer.commit.offset.period", 15),
-    CONSUMER_COMMIT_OFFSET_QUEUES_SIZE("consumer.commit.offset.queues.size", 500_000),
+    CONSUMER_RECEIVER_POOL_TIMEOUT("consumer.receiver.pool.timeout", 100),
+    CONSUMER_RECEIVER_READ_QUEUE_CAPACITY("consumer.receiver.read.queue.capacity", 1000),
+
+    CONSUMER_COMMIT_OFFSET_PERIOD("consumer.commit.offset.period.ms", 15 * 1000),
+    CONSUMER_COMMIT_OFFSET_QUEUES_SIZE("consumer.commit.offset.queues.size", 1000),
```

Comment:
I'm a bit afraid introducing some dynamic behaviour here will not be good for us (and hard to debug). Bounded queue has array underneath, and thus growing it will not be so easy and might kill us in case of burst of traffic, like consuming lag from a lot of subscriptions.

I don't mean using it concurrently, but simply sth like sending "Commit" signal to Consumer? I don't know..